src/App.tsx

```diff
@@ -1,13 +1,19 @@
 import React, { useState } from 'react';
+import { getLogger } from 'loglevel';
 import { AppBar, Container, Toolbar, Grid, Paper, CssBaseline, makeStyles, Typography } from '@material-ui/core';
+import AudioDeviceTestWidget from './AudioDeviceTestWidget/AudioDeviceTestWidget';
 import BrowserCompatibilityWidget from './BrowserCompatibilityWidget/BrowserCompatibilityWidget';
 import CopyResultsWidget from './CopyResultsWidget/CopyResultsWidget';
 import { Device } from 'twilio-client';
 import { getJSON } from './utils';
+import { APP_NAME, LOG_LEVEL } from './constants';
 import NetworkTestWidget from './NetworkTestWidget/NetworkTestWidget';
 import ResultWidget from './ResultWidget/ResultWidget';
 import SummaryWidget from './SummaryWidget/SummaryWidget';
 
+const log = getLogger(APP_NAME);
+log.setLevel(LOG_LEVEL);
```

Comment:
`log` is defined but never used. Should this be here?

Reply:
Oh, does this create the logger that's used in `useTestRunner`?

Reply:
Yes, `getLogger` is app-wide.